src/convex/ConvexAvatarMultiToken.sol

```diff
@@ -446,15 +452,14 @@ contract ConvexAvatarMultiToken is BaseAvatar, ConvexAvatarUtils, PausableUpgrad
     /// @dev This is a failsafe to handle rewards manually in case anything goes wrong (eg. rewards need to be sold
     ///      through other pools)
     function claimRewardsAndSendToOwner() external onlyOwner {
-        address ownerCached = owner();
         // 1. Claim CVX, CRV & FXS rewards
         (uint256 totalCrv, uint256 totalCvx, uint256 totalFxs) = claimAndRegisterRewards();
 
         // 2. Send to owner
-        CRV.safeTransfer(ownerCached, totalCrv);
-        CVX.safeTransfer(ownerCached, totalCvx);
+        CRV.safeTransfer(msg.sender, totalCrv);
+        CVX.safeTransfer(msg.sender, totalCvx);
```

Comment:
QUESTION: Why not check for 0 for all others?

Reply:
i guess because it is assumed that if a crv token is emitted, it is matched with a cvx. hence there will always be crv&cvx  to claim.

agree with challenging that assumption though; what if cvx is not being emitted? is adding the `if>0` condition worth the gas?

Reply:
6 gas per check, not a consideration gas wise imo

Reply:
initially added only the check for fxs since we may not be always involved in the convex-frax infra, which is the component minting the fxs

nevertheless, its true that the cvx minting ratio is approaching zero, so worth adding for all cases. not much gas overhead. always can happen some unexpected behaviour, even tho crv emissions will outlive us

added in e53920162c1fb3bd9cd9513b6e427a6315756217